MAINTAINERS

```diff
@@ -80,6 +80,7 @@
 			"sam-thibault",
 			"samwhited",
 			"thajeztah"
+			"thompson-shaun",
```

Comment:
```suggestion
			"thajeztah",
			"thompson-shaun"
```

Reply:
LGTM otherwise 😅

Reply:
doh! I initially had it in the wrong list; let me fix

Reply:
should be good now